website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -9,6 +9,394 @@ To-do:
+
+Snowflake Iceberg tables support for the dbt-snowflake adapter is available for three out-of-the-box materializations:
+
+- [Table](/docs/build/materializations#table)
+- [Incremental](/docs/build/materializations#incremental)
+- [Dynamic](#dynamic-tables) 
+
+For now, Iceberg tables require a [behavior flag](/reference/global-configs/behavior-changes) due to performance impact related to using Iceberg tables. Snowflake does not support `is_iceberg` on the  `Show Objects` queries, which dbt depends on for metadata.
+
+To use Iceberg, set the `enable_iceberg_materializations` flag to `True` in your dbt_project.yml:
+
+<File name='dbt_project.yml'>
+
+```yaml
+
+flags:
+  enable_iceberg_materializations: True
+
+```
+
+</File>
+
+If you don't set this behavior flag, you will get an error and dbt will not materialize the table in the Iceberg format.
+
+The following configurations are supported: 
+
+| Field | Type | Required | Description | Sample input |
+|-------|------|----------|-------------|-------|
+|Table Format | String | Yes | Configures the object table format. | `iceberg` |
+|External volume | String | Yes | Specifies the identifier (name) of the external volume where the Iceberg table stores its metadata files and data in Parquet format. If you don’t specify this parameter, the Iceberg table defaults to the external volume for the schema, database, or account. The schema takes precedence over the database, and the database takes precedence over the account.| `my_s3_bucket` |
+|Base location Subpath |String | No | Defines the directory path for Snowflake to write the table data and metadata files. If you change this after creating the table, it will write a new table but leave the existing one. | `jaffle_marketing_folder` |
+
+### Example configuration
+
+To configure an Iceberg table materialization in dbt, refer to the example configuration:
+
+<File name='models/<modelname>.sql'>
+
+```sql
+
+{{
+  config(
+    materialized = "table",
+    table_format="iceberg",
+    external_volume="s3_iceberg_snow",
+    base_location_subpath="jaffle_shop_folder",
+  )
+}}
+
+select * from {{ ref('raw_orders') }}
+
+```
+
+</File>
+
+### Base location 
+
+If you were to create an Iceberg table in the Snowflake query editor, you would be required to provide a `base_location` as an input. dbt abstracts away from that. The default behavior in dbt is to provide a `base_location` string that follows this convention:
+
+`_dbt/{SCHEMA_NAME}/{MODEL_NAME} `
+
+However, dbt allows users to configure a `base_location_subpath` that is empty by default but, when provided, will be concatenated to the end of the previously described pattern for `base_location` string generation. We recommend leaving the `base_location_subpath` field blank by default. The `base_location_subpath` input is always appended to your schema name, and this behavior cannot be overridden. This ensures dbt can differentiate Iceberg model builds by environment.
```

Comment:
didn't mean to edit as heavily as I did but here we are.

note i completely cut this line:
> This ensures dbt can differentiate Iceberg model builds by environment.

```suggestion
#### Base Location Subpath
We recommend using dbt's auto-generated `base_location`. However, if you have a need to customize the resulting `base_location`, dbt allows users to configure a `base_location_subpath`. When specified, it will be concatenated to the end of the previously described pattern for `base_location` string generation.

For example, `config(base_location_subpath="prod")` will generate a `base_location` of the form `_dbt/{SCHEMA_NAME}/{MODEL_NAME}`

A theoretical (not recommended) use case: re-using an `EXTERNAL VOLUME` while still maintaining isolation  across development and production environments. This is not recommended as storage permissions should configured on the external volume and underlying storage, not paths that can be modified by any analytics engineer.
```